docs/writing-docs/skillmaps.md

```diff
@@ -0,0 +1,222 @@
+* **secondarycolor**: hex color value for unlocked nodes
+* **tertiarycolor**: hex color value for background
+* **highlightcolor**: hex color value for the selected node border and unlocked reward nodes
+* **alternatesources**: ???
+
+A property list from a skillmap in MakeCode Arcade provides an example:
+
+```markdown
+# game-maker-guide
+
+* name: Game Maker Guide
+* description: Level up your game making skills by completing the tutorials in this guide.
+* infoUrl: skillmap/educator-info/int-map-info
+* bannerUrl: /static/skillmap/platformer/activity4.png
+* backgroundurl: /static/skillmap/space/game-maker-guide-background.png
+* primarycolor: #2EA9B0
+* secondarycolor: #F392BD
+* tertiarycolor: #83C252
+* highlightcolor: #FAED28
+* alternatesources: github:https://github.com/microsoft/pxt-skillmap-sample/skillmap.md
+```
+
+The following skillmap has 3 different paths defined. The panel on the right shows the name of the map with a description and a banner image.
+
+![Full skillmap graphic](/static/skillmaps/skillmap.jpg)
+
+### Path properties
+
+The path properties are in the second-level (`##`) headings using the paths's name, or its _path-id_.
+
+* **path-id**: the identification name of the path
+
+```
+## space
+```
+
+After the heading is the path property list.
+
+* **name**: path display name
+* **description**: the path description
+* **completionUrl**: the URL path to a completion document (i.e. the "Certificate of Completion")
+
+Here's an example of a property list set for a path:
+
+```
+## space
+
+* name: Design a Space Explorer
+* description: Let's explore the depths of space! We'll design a vessel for space travel, add some enemies, and populate the universe with planets.
+* completionUrl: /static/skillmap/certificates/design-a-space-explorer.pdf
+```
+
+### ~ hint
+
+#### Selecting a path
+
+When selecting a path in a skillmap, the focus goes to the activity node. That node's properties are displayed along with the path description.
+
+### ~
+
+The selected path shows in the skillmap panel along with the property info from the activity node in focus.
+
+![Individual skillmap path](/static/skillmaps/skillmap-path.jpg)
+
+### Activities (nodes)
+
+An activity node sets a current activity and connects it to the next one. Activities are created as [tutorials](/writing-docs/tutorials) and linked together in the node properties.
+
+The activity node properties are under a third-level (`###`) heading which has the nodes's name, or its _node-id_. The activity nodes are grouped sequentially under their associated path section.
+
+* **node-id**: the identification name of the node
+
+```
+### space-activity3
+```
+
+The node properties are specified under the `node-id` heading.
+
+* **name**: the display name of the activity node
+* **description**: description of the activity
+* **type**: the type of node, `tutorial` or `certificate`. Currently only the `tutorial` type is used
```

Comment:
hmm so basically the setup is: you can specify the `kind` of the node, and each kind  will have "types" (subtypes?) within it. so the default kind of `activity` would theoretically have a set of available types like "tutorial", "game", "design"--but right now we only have "tutorial". and the `completion` kind would have types like "certificate", "assets", "site-theme" (types of prizes), but right now we've only implemented "certificate". 

i think we can leave the `layout` kind out completely for now!